checkov/common/parallelizer/parallel_runner.py

```diff
@@ -38,7 +38,10 @@ def func_wrapper(original_func, items_group, connection):
 
         for process, parent_conn, group_len in processes:
             for _ in range(group_len):
-                yield parent_conn.recv()
+                try:
+                    yield parent_conn.recv()
+                except EOFError:
+                    pass
```

Comment:
@rotemavni When is this error thrown?

Reply:
@nimrodkor 
It's being thrown by recv():
Return an object sent from the other end of the connection using send(). Blocks until there is something to receive. Raises EOFError if there is nothing left to receive and the other end was closed.
https://docs.python.org/3/library/multiprocessing.html